src/App.js

```diff
@@ -1,34 +1,63 @@
-import React, { useCallback, useReducer, useEffect } from "react";
-import "./App.css";
-import { reducer, init } from "./Reducer";
+/** @jsx jsx */
+import { jsx, css } from '@emotion/core';
+import styled from '@emotion/styled';
+import { useCallback, useReducer, useEffect } from 'react';
+import './App.css';
+import { reducer, init } from './Reducer';
 import {
   addTaskAct,
   changeNameAct,
   checkAct,
   initAct,
   removeAct,
   viewAct
-} from "./Actions";
+} from './Actions';
 
 const my_todo_list = [
-  { id: 1, name: "Get Milk", done: false },
-  { id: 2, name: "Go to Gym", done: false },
-  { id: 3, name: "Call Mom", done: false },
-  { id: 4, name: "Work", done: false },
-  { id: 5, name: "Send Email", done: false }
+  { id: 1, name: 'Get Milk', done: false },
+  { id: 2, name: 'Go to Gym', done: false },
+  { id: 3, name: 'Call Mom', done: false },
+  { id: 4, name: 'Work', done: false },
+  { id: 5, name: 'Send Email', done: false }
 ];
 const TodoItem = ({ id, name, remove, done, check }) => (
-  <div style={{ width: null }}>
-    <input checked={done} type="checkbox" onClick={check} />
+  <div css={css`margin:10px;`} onClick={check}>
```

Comment:
Use https://emotion.sh/docs/styled components instead of passing css, define styled components in another file (`App.styles.js` for example) and then import and use them, for example this component should be named `Container`, the input inside this div can be named `NewItemInput` or simply `Input`, the button `AddButton`, etc.